### ISSUE TITLE
Can't choose text+ template. Despite using a script from pull request

### ISSUE THREAD

**Issue by wetyw** [tmoroney/auto-subs#67]
I've ran into an issue with autosubs when I updated to the newest beta release. 

I've had this problem with the script not opening.  
![Zrzut ekranu 2024-08-13 173150](https://github.com/user-attachments/assets/9dce0ee8-1fd6-44ca-a75b-9265d583ef69)

I got the solution from the discord server but now I can't choose the text+ template. 
( Solution )
![Zrzut ekranu 2024-08-15 022143](https://github.com/user-attachments/assets/b045bd4f-81da-467e-8030-8967c2df57b9)

I've seen that the pull request did make it work but not for me. I tried to merge two solutions into one but it didn't work out either.

**Comment by h4k4s4m** [comment]
> I'm on version 19.0b6 and it seems that the functions like GetName() ,GetClipProperty('Type') ,GetClipProperty('Clip Name') all return none for Text+ and other title items, so the code is completely broken as of now. Not really sure where to go from here

Yup, 19.0b6 indeed breaks the script and you get some "nonetype" bullcrap.

You'll need to uninstall and reinstall b5 

# Windows
https://www.blackmagicdesign.com/support/download/67c93e8934524d0096517717358c9370/Windows
# Mac
https://www.blackmagicdesign.com/support/download/67c93e8934524d0096517717358c9370/Mac%20OS%20X
# Linux
https://www.blackmagicdesign.com/support/download/67c93e8934524d0096517717358c9370/Linux


Additionally latest version of pytorch also breaks the script with some DLL not being found even though its physically in the right location. Fix this by installing this version

# ROCM 6.0 (Linux only)
pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/rocm6.0
# CUDA 11.8
pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/cu118
# CUDA 12.1
pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/cu121
# CPU only
pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/cpu


Don't update your shizite for now, folks

_Originally posted by @h4k4s4m in https://github.com/tmoroney/auto-subs/issues/64#issuecomment-2295094506_

**Comment by wetyw** [comment]
> ```
>           > I'm on version 19.0b6 and it seems that the functions like GetName() ,GetClipProperty('Type') ,GetClipProperty('Clip Name') all return none for Text+ and other title items, so the code is completely broken as of now. Not really sure where to go from here
> ```
> 
> Yup, 19.0b6 indeed breaks the script and you get some "nonetype" bullcrap.
> 
> You'll need to uninstall and reinstall b5
> 
> # Windows
> https://www.blackmagicdesign.com/support/download/67c93e8934524d0096517717358c9370/Windows
> 
> # Mac
> https://www.blackmagicdesign.com/support/download/67c93e8934524d0096517717358c9370/Mac%20OS%20X
> 
> # Linux
> https://www.blackmagicdesign.com/support/download/67c93e8934524d0096517717358c9370/Linux
> 
> Additionally latest version of pytorch also breaks the script with some DLL not being found even though its physically in the right location. Fix this by installing this version
> 
> # ROCM 6.0 (Linux only)
> pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/rocm6.0
> 
> # CUDA 11.8
> pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/cu118
> 
> # CUDA 12.1
> pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/cu121
> 
> # CPU only
> pip install torch==2.3.1 torchvision==0.18.1 torchaudio==2.3.1 --index-url https://download.pytorch.org/whl/cpu
> 
> Don't update your shizite for now, folks
> 
> _Originally posted by @h4k4s4m in [#64 (comment)](https://github.com/tmoroney/auto-subs/issues/64#issuecomment-2295094506)_

I don't understand. The link leads to Fusion Studio download. It's seperate software. Did you mean reinstalll Resolve b5?

**Comment by h4k4s4m** [comment]
Yes sorry, I fixed the links.

I uninstalled 9b6 and reinstalled 9b5 but you may lose things like keyboard shortcuts and stuff.

My projects seem to all be there so if you don't use a lot of custom config you might be clear to just uninstall resolve plus control panel and reinstall with the links I provided (or find what you need on https://www.blackmagicdesign.com/support/ and scroll down the widget on the bottom left)

**Comment by wetyw** [comment]
Yeah. Installing b5 fixed the issue for me. I needed to do some backups. The loss of shortcuts is painful but I'll live with it.
